package.json

```diff
@@ -2,56 +2,43 @@
   "name": "bitski",
   "description": "Bitski JavaScript SDK",
   "license": "MIT",
-  "main": "dist/bitski.umd.js",
-  "module": "dist/bitski.es5.js",
+  "main": "dist/lib/bitski.js",
   "typings": "dist/types/bitski.d.ts",
   "repository": {
     "type": "git",
-    "url": "https://github.com/BitskiCo/bitski-js"
+    "url": "https://github.com/BitskiCo/bitski-js-sdk"
   },
-  "version": "0.1.10",
+  "version": "0.2.0",
   "scripts": {
     "lint": "tslint 'src/**/*.ts'",
-    "build": "tsc && rollup -c rollup.config.ts && webpack --mode production && typedoc",
-    "prepack": "npm run build",
-    "dev": "rollup -c rollup.config.ts -w",
+    "build": "tsc && typedoc",
+    "prepack": "npm install && tsc",
```

Comment:
curious about these build scripts. so we're no longer using webpack or rollup?

Reply:
@pixelmatrix I switched to vanilla tsc for debugging, and found it to work a lot better. Node seems to take care of these things automatically.